webapp/src/views/ResetPassword.vue

```diff
@@ -79,19 +79,30 @@
                                             tabindex="3"
                                     />
                                   </v-col>
```

Comment:
hint for the audio alternative is missing (it's in the sign-up form)

Reply:
done